libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -9,34 +9,48 @@ import {
 } from '@tmo/books/data-access';
 import { FormBuilder } from '@angular/forms';
 import { Book } from '@tmo/shared/models';
-import { Observable } from 'rxjs';
+import { debounceTime, distinctUntilChanged, takeUntil } from 'rxjs/operators';
+import { Subject, Observable } from 'rxjs';
 
 @Component({
   selector: 'tmo-book-search',
   templateUrl: './book-search.component.html',
-  styleUrls: ['./book-search.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush
+  styleUrls: ['./book-search.component.scss']
 })
-export class BookSearchComponent implements OnInit {
-  books$: Observable<ReadingListBook[]>;
+export class BookSearchComponent implements OnInit, OnDestroy {
+  books$: Observable<ReadingListBook[]> = this.store.select(getAllBooks);
+  private destroyedFormValues$: Subject<void> = new Subject();
```

Comment:
not a good name for this IMO. What if in future you have another subscription in this component file that needs to be closed on tear down? Use a more generic name for this subject

Reply:
I kept separate subject specific to form value changes subscription. That's the reason I named it as destroyedFormValues$. For the new subscription it will be new variable.
Please suggest.